AsyncDisplayKitTests/ASDisplayNodeTests.m

```diff
@@ -1989,7 +1980,7 @@ - (void)testDidEnterFetchDataIsCalledWhenNodesEnterFetchDataRange
 
 - (void)testDidExitFetchDataIsCalledWhenNodesExitFetchDataRange
 {
-  ASTestDisplayNode *node = [[ASTestDisplayNode alloc] init];
+  ASTestDisplayNode *node = [[[ASTestDisplayNode alloc] init] autorelease];
```

Comment:
I only changed and/or converted to the use of `-autorelease` where it was necessary to fix the memory leaks that were found.

I noticed that other parts of the code were using the explicit `alloc` / `release` pattern, but I didn't touch those.  IMHO, "the right thing" would be to go through the `-fno-objc-arc` unit test code and convert everything to use `-autorelease`.